eclipse/src/saros/Saros.java

```diff
@@ -264,25 +273,34 @@ private void setupLoggers() {
      * HACK this is not the way OSGi works but it currently fulfill its
      * purpose
      */
-    final ClassLoader contextClassLoader = Thread.currentThread().getContextClassLoader();
 
     final boolean isDebugMode = Boolean.getBoolean("saros.debug") || isDebugging(); // $NON-NLS-1$
 
-    final String log4jPropertyFile =
-        isDebugMode
-            ? "saros_debug.log4j.properties" //$NON-NLS-1$
-            : "saros_release.log4j.properties"; //$NON-NLS-1$
-
     try {
-      // change the context class loader so Log4J will find the appenders
-      Thread.currentThread().setContextClassLoader(Saros.class.getClassLoader());
+      File mainLogDir = getStateLocation().toFile();
+      final String logDir = mainLogDir + File.separator + "SarosLogs";
```

Comment:
Why do you name it SarosLogs instead of just log ?

../.metadata/.plugins/saros.eclipse/SarosLogs 

So this should be the final name ?